### PR TITLE
namespace-admin-home-category

### PR DESCRIPTION
**Pull request by phamhuycuong**

### CODE REVIEW

File: app/controllers/admin/categories_controller.rb [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+  def new
+    @category = Category.new
+  end
+
+  def edit
+  end
+
+  def create
+    @category = Category.new category_params
+    if  @category.save
```
Comment (khanhkhanh) [inline app/controllers/admin/categories_controller.rb]: 2 spaces after if

File: app/views/admin/categories/_show_categories.html.erb [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        <%= link_to I18n.t(:destroy), admin_category_path(category), method: :delete, data: {confirm: I18n.t(:are_you_sure)} %>
+      </td>
+    </tr>
+  </table>
+  <hr>
+<% end %>
+<table class="w100">
+  <tr>
+    <td align="left"><%= will_paginate @categories, renderer: BootstrapPagination::Rails %></td>
+    <td align="right"><%= link_to_if(current_user.admin?, I18n.t(:new_category), new_admin_category_path){}%></td>
```
Comment (khanhkhanh) [inline app/views/admin/categories/_show_categories.html.erb]: Remove the outermost (), and add a space after {}

File: app/views/admin/categories/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1><%= I18n.t(:edit_category) %></h1>
```
Comment (khanhkhanh) [inline app/views/admin/categories/edit.html.erb]: Remove ()

File: app/views/admin/categories/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,6 @@
+<h1><%= I18n.t(:listing_categories) %></h1>
```
Comment (khanhkhanh) [inline app/views/admin/categories/index.html.erb]: Remove ()

File: app/views/admin/categories/new.html.erb [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<h1><%= I18n.t(:new_category) %></h1>
```
Comment (khanhkhanh) [inline app/views/admin/categories/new.html.erb]: Remove ()

File: config/routes.rb [hunk 6]
```diff
@@ -8,10 +8,17 @@
...
+    resources :categories
+    resources :words
+    resources :users
+  end
+
+  resources :words, only: [:index]
+  resources :users, except: [:destroy]
 
-  resources :categories do
+  resources :categories, only: :index do
```
Comment (khanhkhanh) [inline config/routes.rb]: ??? new, create ???

**Comment by dieunb** [comment]
OK

**Pull request merged**